File: Python Word Examples/TextRangesApiDemo/addTextRange.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TextRangesApi(configuration)

file_name = 'Template.docx'
paragraph_path = 'Section/0/Body/0/Paragraph/0'
text = 'AddTextRange'
extra_text = { 'index_in_paragraph': 0,  'folder': "input",'dest_file_path': 'output/AddTextRange.docx'}
result =api.add_text_range(file_name, paragraph_path, text, **extra_text)

File: JavaScript Word Examples/TextRangeApiDemo/addTextRange.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TextRangesApi(apiClient);

    var filename = "AddTextRange.docx";
    var paragraphPath = "Section/0/Body/0/Paragraph/0";
    var destFilePath = "output/AddTextRange_out.docx";
    var text = "AddTextRange";
    var opt = {
        folder: 'input',
        indexInParagraph: null,
        storage: '',
        password: ''
    }    
    instance.addTextRange(filename, paragraphPath, text, destFilePath, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();